libnopegl/meson.build

```diff
@@ -130,6 +130,7 @@ project_args = cc.get_supported_arguments([
   '-Wno-sign-conversion',
   '-fmacro-prefix-map=@0@/='.format(trim_prefix),
   '-fno-math-errno',
+  '-Wundef',
```

Comment:
probably makes sense in ngl tools as well?

Also, can you add it below the last `-W*` so that warning flags are together?

Reply:
Done